client/extensions/woocommerce/state/ui/orders/edits/test/reducer.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import reducer from '../reducer';
+import {
+	WOOCOMMERCE_UI_ORDERS_CLEAR_EDIT,
+	WOOCOMMERCE_UI_ORDERS_EDIT,
+} from 'woocommerce/state/action-types';
+
+describe( 'reducer', () => {
+	it( 'should store the currently editing order', () => {
+		const action = {
+			type: WOOCOMMERCE_UI_ORDERS_EDIT,
+			siteId: 123,
+			order: {
+				id: 40,
+				billing: {
+					first_name: 'Joan',
+				}
+			},
+		};
+		const newState = reducer( undefined, action );
+		expect( newState ).to.eql( {
+			currentlyEditingId: 40,
+			changes: {
+				billing: {
+					first_name: 'Joan',
+				}
+			},
+		} );
+	} );
+
+	it( 'should update the order when it\'s changed', () => {
+		const action = {
+			type: WOOCOMMERCE_UI_ORDERS_EDIT,
+			siteId: 123,
+			order: {
+				id: 40,
+				billing: {
+					first_name: 'Joan',
+					last_name: 'Watson',
+				}
+			},
+		};
+		const originalState = deepFreeze( {
+			currentlyEditingId: 40,
```

Comment:
Should currentlyEditingId be null instead to best exercise the reducer for action type WOOCOMMERCE_UI_ORDERS_EDIT?

Reply:
The test above this one checks that `currentlyEditingId` is correctly set from the empty state - and it doesn't make sense that there would be existing edits in the state without having `currentlyEditingId` defined.